src/SegmentedArc.js

```diff
@@ -41,7 +42,7 @@ export const SegmentedArc = ({
   const totalSpacing = totalSpaces * spaceBetweenSegments;
 
   const arcSegmentDegree = (arcDegree - totalSpacing) / totalArcs;
-  const arcsStart = 90 - arcDegree / 2;
+  const arcsStart = angleStart || 90 - arcDegree / 2;
```

Comment:
Think it's worth documenting any guidance on realistic values here? Or does it feel pretty obvious. Code makes sense, just thinking aloud

Reply:
Example of 720 angleStart:

<img width="156" alt="image" src="https://github.com/shipt/segmented-arc-for-react-native/assets/24547965/e82a33ef-c945-42f9-94de-f85bda485415">

Reply:
What do you think about `arcStartDegree`? Also, would you be able to update docs too along with the previously added `arcDegreeScale` property? Could be done separately. https://github.com/shipt/segmented-arc-for-react-native?tab=readme-ov-file#-props